### DISCUSSION TITLE
Welcome to openlittermap-web Discussions!

### DISCUSSION THREAD

**Discussion by xlcrr** [OpenLitterMap/openlittermap-web#71] [General]
## 👋  Welcome!

We are going to use Discussions as a place to connect with other members of our community.

Here you will be able to

  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Discuss various aspects of our platform and help us plan for the future 

  To get started, comment below with an introduction of yourself and tell us about what do you think about our work so far, and how would you like to contribute more? Who are you, where are you from?

**Comment by FredMark** [comment]
I'm Fred,
I am a litterpicker from the Netherlands, i have been picking litter for some years now and after some time i started taking pictures of the litter as well, because by doing so, i could make an even bigger impact.
My biggest success so far was picking litter along to work and finding plastic gloves everyday.
I picked them, created data, confronted the holdingcompany (who did not respond) and confronted the company where i found them.
It took me about a year, but after that, they gave in and changed their way of distributing gloves to the clients and i have not found a single one after that.
This would not have been possible without taking pictures and using the data that i was collecting.
And that is exactly why i support OLM, because i believe in this data and i love the fact that it is open-source.
But, having said that, i pick a lot of litter, so i also want to be able to create this data 'as fast as possible'. To me, every mouseclick or keyboard-push counts, so i would ask every developer to think about every button that they create and ask themselves 'is this one necessary, can i think of a way that speeds up uploading and tagging'.
All this to create a software solution that litterpickers love to use but also to spend as little time as possible on so they can go out and pick litter and to create even more data :-)
